config/crds/app_v1beta1_application.yaml

```diff
@@ -189,6 +189,75 @@ spec:
           type: object
         status:
           properties:
+            components:
+              items:
+                properties:
+                  group:
+                    type: string
+                  kind:
+                    type: string
+                  link:
+                    type: string
+                  name:
+                    type: string
+                  pdb:
```

Comment:
Should this be the short name? I think using the full name would make more sense.

Reply:
can change it.

Reply:
done